### ISSUE TITLE
Restore the "Shutdown complete" log from collector

### ISSUE THREAD

**Issue by astencel-sumo** [open-telemetry/opentelemetry-collector#5625]
**Is your feature request related to a problem? Please describe.**

The collector v0.52.0 and before used to write an INFO log on shutdown that said "Shutdown complete".
This was useful e.g. in testing, to automatically verify that the collector has shut down completely.

```console
2022-07-04T12:05:28.513+0200	info	service/collector.go:278	Shutdown complete.
```

In v0.53.0 and later, this log has been [removed](https://github.com/open-telemetry/opentelemetry-collector/commit/1aa0e5ab3667721f282b557346779cb4ebec4903#diff-780b10647e2c9aaf7a1a80898c979f5018106a37a92b0d2ce4728017b28be2f6L278).

Now I don't seem to have a clear way of verifying that the collector has shut down.

**Describe the solution you'd like**

I propose to restore this single log. Hopefully this is a simple and painless change. 🤞

**Comment by astencel-sumo** [comment]
Here's the simple pull request implementing the change: https://github.com/open-telemetry/opentelemetry-collector/pull/5626.

I was worried that access to logger is not possible at this stage with the refactoring made in service/collector.go, but it seems to run fine for now.

**Comment by bogdandrutu** [comment]
Fixed in https://github.com/open-telemetry/opentelemetry-collector/pull/5913

**Comment by astencel-sumo** [comment]
> Fixed in #5913

Thanks a lot Bogdan for the update, it's good to know the "Shutdown complete" log is back.
